benchmarks/bge_m3/src/scripts/bench_onnx_cpu_stage_breakdown.py

```
@@ -116,6 +144,15 @@ def main() -> None:
     if args.warmup < 0:
         parser.error(f"--warmup must be >= 0 (got {args.warmup})")
 
+    if args.precision == "int8" and (
+        not args.reference_model_dir
+        or Path(args.reference_model_dir) == model_dir
```

Ruff (0.15.14)

[error] Undefined name `model_dir`

(F821)